components/accordion/package.json

```
@@ -19,12 +19,12 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.0",
-    "@spectrum-css/vars": ">=9.0.0"
+    "@spectrum-css/tokens": "^8.1.0"
```

```suggestion
    "@spectrum-css/tokens": ">=8 <=9"
```

Should we make this more permissive setting the standard for peerDeps?